### PR TITLE
Adds new integration [flexopus/flexopus-hass-sensor]

### PR DESCRIPTION
**Pull request by sebinemeth**
## Checklist

- [x] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [x] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [x] The actions are passing without any disabled checks in my repository.
- [x] I've added a link to the action run on my repository below in the links section.
- [x] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: [v1.0.1](https://github.com/flexopus/flexopus-hass-sensor/releases/tag/v1.0.1)
Link to successful HACS action (without the `ignore` key): [https://github.com/flexopus/flexopus-hass-sensor/actions/runs/6585659304/job/17892512931](https://github.com/flexopus/flexopus-hass-sensor/actions/runs/6585659304/job/17892512931)
Link to successful hassfest action (if integration): [https://github.com/flexopus/flexopus-hass-sensor/actions/runs/6585659304/job/17892513169](https://github.com/flexopus/flexopus-hass-sensor/actions/runs/6585659304/job/17892513169)

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -311,6 +311,7 @@
   "finity69x2/nws_alerts",
   "firstof9/ha-openei",
   "FL550/dwd_weather",
+  "flexopus/flexopus-hass-sensor",
```
Comment (ludeeus) [inline integration:314]: PR is not editable.
You need to adjust your branch settings.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 1 above)
Comment (sebinemeth) [reply integration:332]: @ludeeus thanks for the review, updated the branch and added you with write access

Comment (ludeeus) [reply integration:332]: That is wrong. I will not accept that invitation.
That is not how you make PRs editable if you can not do it.
I suggest you do a new PR not from the "master" branch.
<https://hacs.xyz/docs/publish/include#before-submitting>
> In your fork of hacs/default, create a new branch for your changes from the master branch. Do not use the master branch directly for your changes.

Comment (sebinemeth) [reply integration:332]: thanks for the clarification, and sorry for the confusion! i will get back with a modified PR

**Comment by sebinemeth** [comment]
I have created a new PR: https://github.com/hacs/default/pull/2282

FYI, cross organization "Allow edits from maintainers" is broken: https://github.com/orgs/community/discussions/5634

**Pull request closed without merging**